Samples/iOS-Swift/iOS-SwiftUITests/LaunchUITests.swift

```diff
@@ -11,15 +11,14 @@ class LaunchUITests: XCTestCase {
         app.launch()
         
         waitForExistenceOfMainScreen()
-        checkSlowAndFrozenFrames()
```

Comment:
Why won't we check this for every test case anymore?

Reply:
Because we would need to be in the last tab after every test.
Its this relevante? 
IMO we just need to check with one test whether slow and frozen frames are being recorded.

Reply:
Fair enough, one test check should be fine 👍🏻